apps/client/src/app/pages/portfolio/transactions/transactions-page.html

```diff
@@ -3,6 +3,7 @@
     <div class="col">
       <h3 class="d-flex justify-content-center mb-3" i18n>Activities</h3>
       <gf-activities-table
+        pageSize="50"
```

Comment:
Remove line (see comment in `libs/ui/src/lib/activities-table/activities-table.component.ts`)

Reply:
Done